addons/website/tests/test_ui.py

```diff
@@ -216,3 +216,8 @@ def test_11_website_snippet_background_edition(self):
             'mimetype': 'image/png',
         })
         self.start_tour('/', 'snippet_background_edition', login='admin')
+
+    def test_12_edit_translated_page_redirect(self):
+        self.env['base.language.install'].create({'lang': 'nl_NL'}).lang_install()
```

Comment:
`_activate_lang()` as I hinted you on my previous message, see `test_lang_url` test file..
This is overkill, it is loading translation and returning an action etc but you only need to activate the lang